apps/charterafrica/payload.config.ts

```diff
@@ -73,6 +76,11 @@ const adapter = s3Adapter({
 
 export default buildConfig({
   serverURL: appURL,
+  editor: slateEditor({}),
+  db: mongooseAdapter({
+    url: process.env.MONGO_URL,
+    migrationDir: "./migrations/v2",
```

Comment:
Shouldn't this be `src/migrations`? Or are they migrations for `just` `v1` to `v2` migration.

Reply:
@kilemensi there exists a folder called migrations with a file for migrations we implemented in v1. To avoid conflicts, I thought it best to have v2 migrations in a folder called v2.

Reply:
... nah, all migrations should be in one folder so they get applied sequentially @koechkevin.

Unless you think the old migration won't run in the version (which would mean we need to decided what to do with them).

Reply:
@kilemensi Yes, the old migration won't run in the new version. Meaning we have 2 options with it, delete the file if the migrations have already been applied or  rewrite the script to ensure it runs in the current version.